src/items/modules.md

```diff
@@ -65,27 +65,22 @@ not have a `path` attribute, the path to the file mirrors the logical [module
 path].
 
 r[items.mod.outlined.search]
-Ancestor module path components are directories, and the module's
-contents are in a file with the name of the module plus the `.rs` extension.
+Ancestor module path components are directories, and the module's contents are in a file with the
+name of the module plus the `.rs` extension. Alternatively, the module's contents can also be in a
+file called `mod.rs` in a directory with the name of the module. It is not allowed to have both
+`$name.rs` and `$name/mod.rs`.
+
 For example, the following module structure can have this corresponding
 filesystem structure:
 
 Module Path               | Filesystem Path  | File Contents
 ------------------------- | ---------------  | -------------
 `crate`                   | `lib.rs`         | `mod util;`
-`crate::util`             | `util.rs`        | `mod config;`
+`crate::util`             | `util.rs` *or* `util/mod.rs` | `mod config;`
 `crate::util::config`     | `util/config.rs` |
```

Comment:
This says "can have", so I realize that it doesn't have to show every possibility. However, it seems like *either* it should consistently show only one case (dropping the addition of ``` *or* `util/mod.rs` ```, or it should show the second case for `crate::util::config` as well (adding ``` *or* `util/config/mod.rs` ```).

I would personally favor the former, but if you want to add the `mod.rs` version to this table, please add it to both lines.